roboteam_ai/test/SkillTests/GoToPosTest.cpp

```diff
@@ -45,6 +43,13 @@ TEST(GoToPos, GoToPosTest) {
 
     EXPECT_EQ(goToPos.update(), bt::Leaf::Status::Running);
 
+    robot.pos.x = 5.0;
+    robot.pos.y = 6.0;
+
+    worldMsg.us[0] = robot;
+    rtt::ai::World::set_world(worldMsg);
+
+    EXPECT_EQ(goToPos.update(), bt::Leaf::Status::Success);
```

Comment:
Maybe you can at least test that the target positions are coming through properly

Reply:
Yes